editor/editor_help.cpp

```diff
@@ -1994,10 +1970,29 @@ void EditorHelp::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_THEME_CHANGED: {
-			if (is_inside_tree()) {
-				_class_desc_resized(true);
-			}
```

Comment:
This was added to prevent crashes as EditorHelp could receive this notification while not in tree - did you check that this is no longer the case?

See #32985 and referenced issues/PRs for details.

Reply:
It's the same thing as https://github.com/godotengine/godot/blob/e06cd2742f8af57b23fa11dec62ba9c4eb544291/editor/plugins/texture_editor_plugin.cpp#L41-L49

`NOTIFICATION_THEME_CHANGED` should now never be called when the node is outside of the tree, so this check is redundant.